frontend/packages/dev-console/src/components/BuildconfigPage.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { match as RMatch } from 'react-router';
+import { BuildConfigsPage } from '@console/internal/components/build-config';
+import DefaultPage from './DefaultPage';
+
+export interface BuildConfigPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+const BuildConfigPage: React.FC<BuildConfigPageProps> = (props) => {
+  const namespace = props.match.params.ns;
+  return (
+    <React.Fragment>
+      <Helmet>
+        <title>Builds</title>
+      </Helmet>
+      {namespace ? (
+        <BuildConfigsPage filterLabel="Resources by name" {...props} />
+      ) : (
+        <DefaultPage title="Build Configs">Select a project to view the build configs</DefaultPage>
```

Comment:
@serenamarie125 says that the page title must be `Builds`.
```suggestion
        <DefaultPage title="Builds">Select a project to view the builds</DefaultPage>
```

This means you need to override the title in `build-config.tsx`

The way this message is written differs from pipelines as it adds here `list of`
`Select a project to view the list of pipelines`